nntrainer/layers/common_properties.cpp

```diff
@@ -58,6 +59,45 @@ bool InputSpec::isValid(const ConnectionSpec &v) const {
   return v.getLayerIds().size() > 0;
 }
 
+/**
+ * @brief unsigned integer property, internally used to parse padding values
+ *
+ */
+class Padding_ : public nntrainer::Property<int> {
+public:
+  using prop_tag = int_prop_tag; /**< property type */
+};
```

Comment:
is it integer?? If it is, then could you add more details in brief for each cases?

Reply:
The `Padding2D` property is string. This class is used only internally inside `common_properties.cpp` to parse padding values.

Reply:
so.. you mean, Padding2D is [ Padding_, Padding_ ] I guess. But If it is.. I guess. it's ok. but.. I prefer to use just < int, int >. It is better in the point of readability.

Reply:
yes, it is std::vector<int, int>, this is like a helper function in a form of class :)